docs/.vuepress/config.js

```diff
@@ -1,37 +1,70 @@
 module.exports = {
-  title: 'Starship',
-  description: 'The cross-shell prompt for astronauts ☄🌌️',
-  head: [
-    ['link', { rel: 'icon', href: '/icon.png' }]
-  ],
+  locales: {
+    "/": {
+      lang: "en-US",
+      title: "Starship",
+      description: "The cross-shell prompt for astronauts"
+    },
+    "/ja/": {
+      lang: "ja-JP",
+      title: "Starship",
+      description: "The cross-shell prompt for astronauts"
+    }
+  },
+  head: [["link", { rel: "icon", href: "/icon.png" }]],
   themeConfig: {
-    logo: '/icon.png',
-    sidebar: [
-      '/',
-      ['/guide/', 'Guide'],
-      ['/config/', 'Configuration'],
-      ['/advanced-config/', 'Advanced Configuration']
-    ],
-    nav: [
-      { text: 'Configuration', link: '/config/' },
-    ],
+    logo: "/icon.png",
     // the GitHub repo path
-    repo: 'starship/starship',
+    repo: "starship/starship",
     // the label linking to the repo
-    repoLabel: 'GitHub',
+    repoLabel: "GitHub",
     // if your docs are not at the root of the repo:
-    docsDir: 'docs',
+    docsDir: "docs",
     // defaults to false, set to true to enable
     editLinks: true,
-    // custom text for edit link. Defaults to "Edit this page"
-    editLinkText: 'Edit this page on GitHub'
+    locales: {
+      "/": {
+        // text for the language dropdown
+        selectText: "Languages",
+        // label for this locale in the language dropdown
+        label: "English",
+        // Custom text for edit link. Defaults to "Edit this page"
+        editLinkText: "Edit this page on GitHub",
+        // Custom navbar values
+        nav: [{ text: "Configuration", link: "/config/" }],
+        // Custom sidebar values
+        sidebar: [
+          "/",
+          ["/guide/", "Guide"],
+          ["/config/", "Configuration"],
+          ["/advanced-config/", "Advanced Configuration"]
+        ],
+      },
+      "/ja/": {
```

Comment:
@unhappychoice Would you be able to translate the strings in this object? 🙂

Reply:
Sure :+1: I wii do that later.